### PR TITLE
Create jake.json

### PR DESCRIPTION
**Pull request by AcedDown**
Please add [jake.json](https://github.com/Aceeeeeeeeeee/register/blob/Aceeeeeeeeeee-patch-1/domains/jake.json)

### CODE REVIEW

File: domains/jake.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "username": "Jake",
```
Comment (MaskDuck) [inline domains/jake.json:3]: your actual github username?

File: domains/jake.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    "email": "hi.cybot@gmail.com",
+    "note": "That email address is not my main email, to make it faster reply to me with a message."
```
Comment (MaskDuck) [inline domains/jake.json:5]: wdym "reply"?

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/jake.json (hunk 1 above)
Comment (Aceeeeeeeeeee) [reply domains/jake.json:3]: fixed the file, check again

File: domains/jake.json (hunk 2 above)
Comment (Aceeeeeeeeeee) [reply domains/jake.json:5]: as for replying in the pull request thread

Comment (MaskDuck) [reply domains/jake.json:5]: we can't have your pr info, so better link more social accounts :)

Comment (Aceeeeeeeeeee) [reply domains/jake.json:5]: Added Discord, that's all I can add for now xD

Comment (MaskDuck) [reply domains/jake.json:5]: mmlol feel free to join our discord to receive notification about the service, also we'll host community event there!

Comment (MaskDuck) [reply domains/jake.json:5]: wait is the file changed? i can't see the file being modified.

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/jake.json (hunk 2 above)
Comment (Aceeeeeeeeeee) [reply domains/jake.json:5]: Weird.. try clicking here https://github.com/Aceeeeeeeeeee/register/blob/Aceeeeeeeeeee-patch-1/domains/jake.json

Comment (MaskDuck) [reply domains/jake.json:5]: image
it is in the `Aceeee-patch-1`, but you are asking us to merge the content in the `main` branch.

Comment (Aceeeeeeeeeee) [reply domains/jake.json:5]: My bad, I have registered in github a long time ago and still didn't manage to learn the basics lmfao

Comment (Aceeeeeeeeeee) [reply domains/jake.json:5]: I deeply apologise if you find me confusing 😅👍

Comment (MaskDuck) [reply domains/jake.json:5]: mm

Comment (Aceeeeeeeeeee) [reply domains/jake.json:5]: *anything else??*

**Pull request closed without merging**